README.md

```diff
@@ -1,11 +1,11 @@
-## Working with Network Proxies
-If you are working behind a corporate firewall, you must provide a sockets based proxy solution in the host OS. [Proxycap](http://www.proxycap.com)  is recommended.
+## Introduction
+This document explains how to use docker containers created to run Yocto Project tools in a Windows or Mac environment.
 ## Configure Containers
 ### Install docker toolbox
 Follow instructions <https://www.docker.com/products/docker-toolbox>
 ### Change default vm settings
-The default Virtual Box VM does not provide enough resources give a good experience when building Yocto images. You will need to allocate 4 physical cores and 8GB of memory to a VM while leaving at two virtual cores and 2GB RAM for host OS.
-* Run the Docker Quickstart Terminal and run the following commands in that terminal.
+The default Virtual Box VM does not provide enough resources to give a good experience when building Yocto images. You will need to allocate 4 physical cores and 8GB of memory to a VM while leaving at two virtual cores and 2GB RAM for host OS.
```

Comment:
2 physical cores is doable, as well as 4GB of RAM. And honestly many laptops don't even have 4 cores, such as my mac. I think recommending something is fine, but this gives the impression that these are requirements.

I also think the part about leaving two virtual cores and 2GB RAM for host os is confusing. That's not going to make sense to most people.